app/editor/src/components/layout/styled/Layout.tsx

```diff
@@ -26,4 +30,10 @@ export const Layout = styled.div`
     display: flex;
     justify-content: center;
   }
+
+  .navbar {
```

Comment:
Keep the logo aligned with the menu.  I don't like the menu design.  Takes up way too much vertical space.   We should make it dynamic at some point.